helpers/password_helper.go

```diff
@@ -4,9 +4,15 @@ import "golang.org/x/crypto/bcrypt"
 
 // HashPassword hashes plain password with bcrypt adaptive hashing algorithm.
 func HashPassword(plainPassword string) (hashedPassword string, err error) {
-	bytePassword := []byte(plainPassword)
-	hashPassword, err := bcrypt.GenerateFromPassword(bytePassword, bcrypt.DefaultCost)
+	hashPassword, err := bcrypt.GenerateFromPassword([]byte(plainPassword), bcrypt.DefaultCost)
 	hashedPassword = string(hashPassword)
 
 	return hashedPassword, err
 }
+
+// CheckMatchPassword checks match password between plain password and hashed password
+func CheckMatchPassword(hashedPassword string, plainPassword string) (err error) {
+	err = bcrypt.CompareHashAndPassword([]byte(hashedPassword), []byte(plainPassword))
+
+	return err
```

Comment:
@Lahphim How about this?
```suggestion
	return bcrypt.CompareHashAndPassword([]byte(hashedPassword), []byte(plainPassword))
```

Reply:
https://github.com/Lahphim/go-crawler-challenge/pull/39/commits/f7ee263032f06878769ff9b0de98072a4b6fecd4